app/src/main/java/de/danoeh/antennapod/menuhandler/FeedItemMenuHandler.java

```diff
@@ -105,6 +105,10 @@ public static boolean onPrepareMenu(Menu menu, FeedItem selectedItem) {
 
         setItemVisibility(menu, R.id.remove_item, fileDownloaded);
 
+        if (selectedItem.getFeed().isLocalFeed()) {
+            setItemVisibility(menu, R.id.share_item, false);
```

Comment:
I think it would be more clear to have local-feed-hiding all in one place (bottom of this method?). Further up the method, there already is some code that hides the website icon, for example.

Reply:
I added another commit to refactor the code as mentioned. Now, there is a common if block for the local-feed-hiding.